packages/@react-spectrum/s2/src/TabsPicker.tsx

```diff
@@ -86,7 +85,11 @@ export interface PickerProps<T extends object> extends
     /** Width of the menu. By default, matches width of the trigger. Note that the minimum width of the dropdown is always equal to the trigger's width. */
     menuWidth?: number,
     /** Density of the tabs, affects the height of the picker. */
-    density: 'compact' | 'regular'
+    density: 'compact' | 'regular',
+    /**
+     * If the tab picker should only display icon and no text for the button label.
+     */
+    iconOnly?: boolean
```

Comment:
Shouldn't this prop name be `isIconOnly`? We've historically uses iconOnly as a variable name. I can't find either being used as a prop name.